build.sbt

```diff
@@ -5,13 +5,13 @@ lazy val core = (project in file("core"))
   .settings(coreSettings:_*)
   .withTestSettings
 
-lazy val gcsfilesystem = (project in file("filesystems/gcs"))
+lazy val gcsFileSystem = (project in file("filesystems/gcs"))
   .settings(gcsFileSystemSettings:_*)
   .withTestSettings
 
 lazy val database = (project in file("database"))
   .settings(databaseSettings:_*)
-  .dependsOn(core % "test->test")
+  .dependsOn(core % "test->test") // TODO: PBE: Remove once PostMVP tag removed from test
```

Comment:
Similar comment which I already know you disagree with, I don't see the point anymore for having todo's tagged as PBE - IMO 'PBE' is a completed project